test/blackbox-tests/test-cases/pkg/helpers.sh

```diff
@@ -80,8 +80,14 @@ solve_project() {
 }
 
 make_lockdir() {
-  mkdir dune.lock
-  cat >dune.lock/lock.dune <<EOF
+  if [ "$#" -eq "0" ]
+  then
+    lockdir="dune.lock"
+  else
+    lockdir="$1"
+  fi
```

Comment:
Not sure if this is POSIX-compliant but you could probably use `${1:-dune.lock}` here.